test/Microsoft.Health.Fhir.Tests.E2E/Rest/HistoryTests.cs

```diff
@@ -55,6 +59,265 @@ public async Task WhenGettingSystemHistory_GivenNoType_TheServerShouldReturnTheA
                 Assert.NotEmpty(readResponse.Resource.Entry);
         }
 
+        [Fact]
+        public void WhenGettingSystemHistory_GivenAValueForSince_TheServerShouldReturnOnlyRecordsModifiedAfterSinceValue()
+        {
+            var since = GetStartTimeForHistoryTest();
+            var sinceUriString = HttpUtility.UrlEncode(since.ToString("o"));
+            Thread.Sleep(1000);
+            _createdResource.Resource.Comment = "Changed by E2E test";
+
+            var updatedResource = Client.UpdateAsync<Observation>(_createdResource).GetAwaiter().GetResult();
```

Comment:
Test method could be `async Task` which will remove the `GetAwaiters` and `.Results`

Reply:
I started with the History tests as async, but that caused the multiple versions (JSON vs XML) to run in parallel and then the number of edits was no longer predictable and the tests would not run reliably.  So I removed all the async code.

Reply:
Makes sense. Wondering if this might work on the HistoryTests class? `[CollectionDefinition(DisableParallelization = true)]`
https://github.com/xunit/xunit/issues/1179

Reply:
Thanks!  That was a good suggestion.  Tests updated with the DisableParallelization enabled.